ios/FluentUI/Core/ColorProviding.swift

```diff
@@ -12,86 +12,52 @@ import UIKit
 @objc(MSFColorProviding)
 public protocol ColorProviding {
     /// If this protocol is not conformed to, communicationBlue variants will be used
-    @objc func brandBackground1(for themeable: FluentThemeable) -> UIColor?
```

Comment:
This is going to have moderate impact on callers - is this something you want in for the 14 release?

Reply:
I am thinking especially of the fallback logic in Loop - this is a good change, but not insignificant.